py/dml/dmllex.py

```diff
@@ -21,7 +21,7 @@
     'AFTER', 'ASSERT', 'BITFIELDS', 'CALL', 'CAST', 'DEFINED', 'ERROR',
     'FOREACH', 'IN', 'IS', 'LAYOUT', 'LOCAL', 'LOG', 'SELECT',
     'SIZEOFTYPE', 'TYPEOF', 'UNDEFINED', 'VECT', '_WARNING', 'WHERE',
-    'EACH', 'SESSION', 'SEQUENCE',
+    'EACH', 'SESSION', 'SEQUENCE', '_',
```

Comment:
It should be 1.4 exclusive, no?  Way I went about it in my WIP work was, in dmllex14:
```
keywords_dml14['_'] = 'DISCARDREF'
tokens.add('DISCARDREF')
```

Reply:
Sure, although somewhat 'tomato tomato'.